design-patterns/behavioral-patterns/src/strategy/ConcreteStrategyMultiply.java

```diff
@@ -0,0 +1,10 @@
+package strategy;
+
+public class ConcreteStrategyMultiply implements Strategy {
```

Comment:
MultiplicationStrategy will fit better as a name. Apply same naming convention for the rest strategies.